### PR TITLE
MAKES data share document migration

### PR DESCRIPTION
**Pull request by alch-msft**
Update MAKES documentation to handle new data provisioning method via data share.

### CODE REVIEW

File: academic-services/graph/get-started-receive-data.md [hunk 1]
```diff
@@ -1,107 +1,13 @@
 ---
-title: Receive Microsoft Academic data
-description: Step-by-step instructions for receiving Microsoft Academic data using Azure Data Share
+title: Receive Microsoft Academic Graph
+description: Step-by-step instructions for receiving Microsoft Academic Graph using Azure Data Share
 ms.topic: get-started-article
 ms.date: 3/23/2021
```
Comment (cyhuang01) [inline academic-services/graph/get-started-receive-data.md:5]: >3/23/2021 [](start = 9, length = 9)

Update ms.date for all modified documents.

File: academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md [hunk 2]
```diff
@@ -22,62 +22,111 @@ To create MAKES index, you will need to format your data into MAKES entity files
 
 ### Define functions to extract MAG data
 
-1. Download `samples/CreateFunctions.usql` from your MAG release to your local drive. <br> From [Azure portal](https://portal.azure.com), go to your MAG Azure Storage account > **Containers > [mag-yyyy-mm-dd] > samples > CreateFunctions.usql > Download**.
+::: moniker range="makes-1.0"
 
-   ![Download CreateFunctions.usql](../graph/media/samples-azure-data-lake-hindex/create-functions-download.png "Download CreateFunctions.usql")
+1. Download `samples/CreateFunctions.usql` from your MAG release to your local drive. 
+
+    From [Azure portal](https://portal.azure.com), go to your MAG Azure Storage account > **Containers > [mag-yyyy-mm-dd] > samples > CreateFunctions.usql > Download**.
```
Comment (cyhuang01) [inline academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md:29]: >From [Azure portal](https://portal.azure.com), go to your MAG Azure Storage account > **Containers > [mag-yyyy-mm-dd] > samples > CreateFunctions.usql > Download**. [](start = 4, length = 164)

From [Azure portal](https://portal.azure.com), go to your MAG Azure Storage account and download **/\<MAG_CONTAINER\>/\<MAG\>/\<YYYY-MM-DD\>/samples/usql/CreateFunctions.usql**.

File: academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md [hunk 3]
```diff
@@ -22,62 +22,111 @@ To create MAKES index, you will need to format your data into MAKES entity files
 
 ### Define functions to extract MAG data
 
-1. Download `samples/CreateFunctions.usql` from your MAG release to your local drive. <br> From [Azure portal](https://portal.azure.com), go to your MAG Azure Storage account > **Containers > [mag-yyyy-mm-dd] > samples > CreateFunctions.usql > Download**.
+::: moniker range="makes-1.0"
 
-   ![Download CreateFunctions.usql](../graph/media/samples-azure-data-lake-hindex/create-functions-download.png "Download CreateFunctions.usql")
+1. Download `samples/CreateFunctions.usql` from your MAG release to your local drive.
```
Comment (cyhuang01) [inline academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md:27]: >samples/CreateFunctions.usql [](start = 13, length = 28)

change "samples/CreateFunctions.usql" to "CreateFunctions.usql"

File: academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md [hunk 4]
```diff
@@ -22,62 +22,111 @@ To create MAKES index, you will need to format your data into MAKES entity files
...
 
 1. The job should finish successfully.
 
-   ![CreateFunctions job status](../graph/media/samples-azure-data-lake-hindex/create-functions-status.png "CreateFunctions job status")
+    ![CreateFunctions job status](../graph/media/samples-azure-data-lake-hindex/create-functions-status.png "CreateFunctions job status")
+
+::: moniker-end
+::: moniker range="makes-3.0"
+
+1. Download `samples/CreateFunctions.usql` from your MAG release to your local drive.
```
Comment (cyhuang01) [inline academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md:48]: >samples/CreateFunctions.usql [](start = 13, length = 28)

change "samples/CreateFunctions.usql" to "CreateFunctions.usql"

File: academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md [hunk 5]
```diff
@@ -154,8 +227,19 @@ You can check the build index job progress using Azure Management Portal. Each j
 
 Now that the custom MAKES index files has been generated. You can follow the same procedures as deploying an index from your MAKES subscription, but replacing the **--MakesIndex** parameter with the path to the output of this example.
 
+::: moniker range="makes-1.0"
+
+```cmd
+Kesm.exe DeployHost --HostName "<makes_instance_host_name>" --MakesPackage "https://<MAS_STORAGE_ACCOUNT>.blob.core.windows.net/makes/<YYYY-MM-DD>/"  --MakesHostImageId "<id_from_create_host_resources_command>" --MakesIndex "https://<MAS_STORAGEACCOUNT>.blob.core.windows.net/dev/makes/<YYYY-MM-DD>/microsoft/index/"
```
Comment (cyhuang01) [inline academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md:233]: ><MAS_STORAGE_ACCOUNT> [](start = 84, length = 21)

why changes needed for makes-1.0?

**Comment by cyhuang01** [comment]
Should delete academic-services\graph\media\receive-data\*.png

File: academic-services/includes/ma-subscription-receive-data.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
+Details step-by-step instructions for receiving Microsoft Academic data using Azure Data Share.
+
+## Prerequisites
+
+Before you can accept Azure Data Share invitation to receive Microsoft Academic data, you must provision a number of Azure resources below. Ensure that all pre-requisites are complete before accepting a data share invitation.
+
+* Set up provisioning of Microsoft Academic Graph to an Azure blob storage account. See [Get Microsoft Academic Graph on Azure storage](get-started-setup-provisioning).
+* A Data Share invitation: An invitation from Microsoft Azure with a subject titled "Azure Data Share invitation from **<yourdataprovider@domain.com>**.
```
Comment (RickRogahn) [inline academic-services/includes/ma-subscription-receive-data.md:8]: is 'from' the right wording here?  Why would the user get an invitation from themselves?

File: academic-services/includes/ma-subscription-receive-data.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+* Register the [Microsoft.DataShare resource provider](https://docs.microsoft.com/azure/data-share/concepts-roles-permissions#resource-provider-registration) in the Azure subscription where you will create a Data Share resource and the Azure subscription where your target Azure data stores are located.
+* Permission to write and add role assignment to the storage account. These permissions exist in the Owner role.
```
Comment (RickRogahn) [inline academic-services/includes/ma-subscription-receive-data.md:10]: assignments

File: academic-services/includes/ma-subscription-receive-data.md [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+
+   If you don't want to accept the invitation, Select **Reject**. 
+
+   ![Accept options](../media/receive-data/accept-options.png "Accept options") 
+
+## Configure received share
+
+Follow the steps below to configure where you want to receive data.
+
+1. Select the **Datasets** tab. Check the box next to the dataset you'd like to assign a destination to. Select **+ Map to target** to choose a target data store. Check both `mag` and `makes` to receive MAG and MAKES data. Or check one dataset to receive only MAG data or only MAKES data.
```
Comment (RickRogahn) [inline academic-services/includes/ma-subscription-receive-data.md:60]: If this is not shared documentation, should we focus on the MAKES selection here?

File: academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md (hunk 5 above)
Comment (alch-msft) [reply academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md:233]: Changing the term "Makes_Storage_Account_Name" to "MAS_STORAGE_ACCOUNT". For makes-3.0/data-share delivery, there's a additional "<MAS_CONTAINER>" param

---
In reply to: [609092229](https://github.com/MicrosoftDocs/microsoft-academic-services/pull/210#discussion_r609092229) [](ancestors = 609092229)

**Review by cyhuang01 — APPROVED** [review]
:shipit:

**Pull request merged**